oonib/oonibackend.py

```diff
@@ -102,60 +102,71 @@
     multiService.addService(http_return_request_helper)
     http_return_request_helper.startService()
 
-# add the tor collector service here
+def getHSEndpoint(endpoint_config):
+    hsdir = os.path.join(torconfig.DataDirectory, endpoint_config['hsdir'])
```

Comment:
The directory should be set by default to either "collector" or "bouncer" to maintain backward compatibility.

Moreover I would expect that if I set the `hsdir` directory to an absolute path it would end up there and not be expanded to a subdirectory of the tor data directory.